tasks/zadanie_15.php

```diff
@@ -17,4 +17,24 @@
 
 $shops = $params[0]; // tej linijki nie ruszamy :)
 
-echo "Najlepsza oferta: Sklep C — 1190 zł";
\ No newline at end of file
+$lowestPrice = null;
+$bestStore = null;
+
+foreach ($shops as $key => $value) {
+  $currentName = $value["store"];
+  $currentPrice = $value["price"];
+
+  if ($lowestPrice == false or $bestStore == false) {
```

Comment:
lepszym porównaniem byłoby :point_down: Ale jako że używasz "słabego" porównania, czy nie sprawdza typów to "działa" :D PS lepiej używać `||` zamiast `or` . OR jest bardziej "tricky" :D można się kiedyś na to naciąć ;) 
```suggestion
  if ($lowestPrice === null || $bestStore === null) {
```